File: GamesTournamentsWeb.Web/ClientApp/models/games/Game.ts
import { ConvertableToJson } from '~/models/ConvertableToJson'

class Game extends ConvertableToJson {
  public id: number
  public name: string
  public description: string
  public genreId: number
  public imageUrl: string

  constructor (id: number, name: string, description: string, genreId: number, imageUrl: string) {
    super()

    this.id = id
    this.name = name
    this.description = description
    this.genreId = genreId
    this.imageUrl = imageUrl
  }
}

export { Game }

File: GamesTournamentsWeb.DataAccess/Models/Tournaments/Game.cs
namespace GamesTournamentsWeb.DataAccess.Models.Tournaments;

public class Game
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public int GenreId { get; set; }
    public Genre Genre { get; set; }
    public string ImageUrl { get; set; }
    
}

